test/e2e/e2e-test/src/test/java/org/apache/skywalking/e2e/PythonE2E.java

```diff
@@ -101,7 +101,7 @@ public void tearDown() {
     @RetryableTest
     void services() throws Exception {
         List<Service> services = graphql.services(new ServicesQuery().start(startTime).end(now()));
-        services = services.stream().filter(s -> !s.getLabel().equals("oap-server")).collect(Collectors.toList());
+        services = services.stream().filter(s -> !s.getLabel().equals("oap::oap-server")).collect(Collectors.toList());
```

Comment:
@kezhenxu94 As a reminder, please consider to don't hardcode this kind of thing in the next-generation e2e framework. This is too tricky.

Reply:
My preference is, so11y test cases should leverage the new `group` field to filter services in https://github.com/apache/skywalking-query-protocol/blob/d3608e86f7c9aaf54aa4ad2c8d2afd09d0680dd6/metadata.graphqls#L72.